owners/src/owners.js

```diff
@@ -123,10 +123,24 @@ class OwnersTree {
   hasOwner(username) {
     const allOwners = this.allRules
       .map(rule => rule.owners)
-      .reduce((left, right) => left.concat(right));
+      .reduce((left, right) => left.concat(right), []);
     return allOwners.includes(username);
   }
 
+  /**
+   * Builds the map from filenames to ownership subtrees.
+   *
+   * @param {string[]} filenames list of changed files.
```

Comment:
The `amphtml` convention for this type is `{Array<string>}`

Reply:
I wasn't able to find a specification for this in either this project or `amphtml`'s `.eslintrc`. Since the entirety of this project currently uses this convention, I propose I file a bug to add a [`jsdoc.preferredTypes` rule](https://www.npmjs.com/package/eslint-plugin-jsdoc#settings-to-configure-check-types-and-no-undefined-types) to our eslint config in both repos, then `eslint --fix` can take care of them all. Thoughts?

Reply:
Fixing this later SGTM.

Reply:
`/bug/issue/s`